plugins/org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/workspace/EclipseProjectConfig.xtend

```diff
@@ -47,6 +52,7 @@ class EclipseProjectConfigProvider implements IProjectConfigProvider {
 class EclipseProjectConfig implements IProjectConfig {
 
 	val IProject project
+	val EclipseProjectConfigProvider projectConfigProvider
```

Comment:
The projectConfigProvider is also part of #toString etc? Not important but it will read strange in the debugger